tests/integration/test.replication.js

```diff
@@ -2395,6 +2395,138 @@ adapters.forEach(function (adapters) {
       });
     });
 
+    it('(#4963) Ensure successful docs are saved but seq not updated if single doc fails to replicate', function (done) {
```

Comment:
typically we use the mocha promise style (i.e. no `done`) and just return a promise from inside this function, but this test looks great, so no worries